sql/migrations/20220205202111_world.sql

```diff
@@ -248,6 +248,43 @@ INSERT INTO `spell_effect_mod` (`Id`, `EffectMiscValue`, `Comment`) VALUES
     (26519, 4, 'Fix Launcher animation for Large Yellow Firework Cluster'),
     (26521, 4, 'Fix Launcher animation for Lucky Lunar Rocket');
 
+-- Thanks to TC brutforced Gameobjects, we have now the real names for the traps (The missing space must be a typo by Blizzard).
+UPDATE `gameobject_template` SET `name`='Port to Moonglade TRAP' WHERE `entry`=180891;
+UPDATE `gameobject_template` SET `name`='Port to StormwindTRAP' WHERE `entry`=180892;
+UPDATE `gameobject_template` SET `name`='Port to Ironforge TRAP' WHERE `entry`=180893;
+UPDATE `gameobject_template` SET `name`='Port to Darnassus TRAP' WHERE `entry`=180894;
+UPDATE `gameobject_template` SET `name`='Port to OrgrimmarTRAP' WHERE `entry`=180895;
+UPDATE `gameobject_template` SET `name`='Port to Undercity TRAP' WHERE `entry`=180896;
+UPDATE `gameobject_template` SET `name`='Port to Thunder Bluff TRAP' WHERE `entry`=180897;
+UPDATE `gameobject_template` SET `faction`='0', `data6`='-1' WHERE `entry` IN (180897,180896,180895,180894,180893,180892,180891);
+
+UPDATE `spell_scripts` SET `comments`='Lunar Teleport Cap: Orgrimmar - Port Alliance Player to Moonglade' WHERE `id`=26414 AND `comments`='Lunar Teleport Cap: Orgrimmar - Send Error to Alliance Player';
+UPDATE `spell_scripts` SET `comments`='Lunar Teleport Cap: Thunderbluff - Port Alliance Player to Moonglade' WHERE `id`=26412 AND `comments`='Lunar Teleport Cap: Thunderbluff - Send Error to Alliance Player';
+UPDATE `spell_scripts` SET `comments`='Lunar Teleport Cap: Undercity - Port Alliance Player to Moonglade' WHERE `id`=26410 AND `comments`='Lunar Teleport Cap: Undercity - Send Error to Alliance Player';
+UPDATE `spell_scripts` SET `comments`='Lunar Teleport Cap: Darnassus - Port Horde Player to Moonglade' WHERE `id`=26409 AND `comments`='Lunar Teleport Cap: Darnassus - Send Error to Horde Player';
+UPDATE `spell_scripts` SET `comments`='Lunar Teleport Cap: Ironforge - Port Horde Player to Moonglade' WHERE `id`=26408 AND `comments`='Lunar Teleport Cap: Ironforge - Send Error to Horde Player';
+UPDATE `spell_scripts` SET `comments`='Lunar Teleport Cap: Stormwind - Port Horde Player to Moonglade' WHERE `id`=26406 AND `comments`='Lunar Teleport Cap: Stormwind - Send Error to Horde Player';
+
+INSERT INTO `spell_effect_mod` (`Id`, `EffectIndex`, `EffectImplicitTargetA`, `Comment`) VALUES
+    (26406, 0, 25, 'Fix for Holiday - Lunar Teleport Cap: Stormwind'),
+    (26408, 0, 25, 'Fix for Holiday - Lunar Teleport Cap: Ironforge'),
+    (26409, 0, 25, 'Fix for Holiday - Lunar Teleport Cap: Darnassus'),
+    (26410, 0, 25, 'Fix for Holiday - Lunar Teleport Cap: Undercity'),
+    (26412, 0, 25, 'Fix for Holiday - Lunar Teleport Cap: Thunderbluff'),
+    (26414, 0, 25, 'Fix for Holiday - Lunar Teleport Cap: Orgrimmar');
```

Comment:
Why is this needed? The spells seemed to already work? I remember i tested your last Lunar Festival PR and there were no problems.

Reply:
it's not needed, i already removed these mods.